packages/botonic-core/src/handoff.ts

```diff
@@ -182,6 +200,12 @@ async function _humanHandOff(
   if (agentId) {
     params.agent_id = agentId
   }
+  if (forceAssignIfNotAvailable !== undefined) {
```

Comment:
and then this could be if(forceAssignIfNotAvailable)

Reply:
@MarcosCA I've answered this comment in the previous one.